app/uk/gov/hmrc/brm/controllers/BirthEventsController.scala

```diff
@@ -70,7 +70,7 @@ trait BirthEventsController extends BRMBaseController {
     // audit the request
     transactionAuditor.transaction(payload, Nil, MatchingResult.noMatch)
 
-    info(CLASS_NAME, "post()", s"Request was not processed due to failing filter(s) $filters. " +
+    info(CLASS_NAME, "post()", s"[failedAtFilter] Request was not processed due to failing filter(s) $filters. " +
```

Comment:
is this is a failed request probably should be at least a warning, maybe an error if it's not a user error